tests/validation/tests/v3_api/test_configmaps.py

```diff
@@ -0,0 +1,489 @@
+    rbac_configmap_delete(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_owner_create():
+    user_token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_create(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_owner_edit():
+    user_token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_edit(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_owner_delete():
+    user_token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_delete(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_member_create():
+    user_token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_create(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_member_edit():
+    user_token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_edit(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_member_delete():
+    user_token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_delete(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_readonly_member_create(remove_resource):
+
+    cluster_owner_token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    project, ns = create_project_and_ns(cluster_owner_token,
+                                        namespace["cluster"],
+                                        random_test_name("rbac-test"))
+    user_token1 = rbac_get_user_token_by_role(PROJECT_READ_ONLY)
+    readonly_user_client = get_project_client_for_token(project, user_token1)
+
+    value = "valueall"
+    keyvaluepair = {"testall": value}
+
+    # Read Only member cannot create config maps
+    with pytest.raises(ApiError) as e:
+        create_configmap(keyvaluepair, readonly_user_client, ns)
+    assert e.value.error.status == 403
+    assert e.value.error.code == 'Forbidden'
+
+
+@if_test_rbac
+def test_rbac_cmap_project_readonly_member_edit(remove_resource):
+
+    cluster_owner_token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    project, ns = create_project_and_ns(cluster_owner_token,
+                                        namespace["cluster"],
+                                        random_test_name("rbac-test"))
```

Comment:
the same as the previous one. We can make use of the existing RBAC framework.

Reply:
Done